solo-chains/runtime/starlight/src/lib.rs

```diff
@@ -1471,10 +1516,9 @@ impl pallet_registrar::RegistrarHooks for StarlightRegistrarHooks {
         // Remove bootnodes from pallet_data_preservers
         DataPreservers::para_deregistered(para_id);
 
-        ServicesPayment::para_deregistered(para_id);
-
         XcmCoreBuyer::para_deregistered(para_id);
          */
+        ServicesPayment::para_deregistered(para_id);
 
         Weight::default()
```

Comment:
It seems we are not using this returned weight. Plus the pallet specific handlers also not returning `Weight`, Can we remove this in future PRs?